src/layers/sds/domain/nhs_accredited_system.py

```diff
@@ -27,7 +27,7 @@ class NhsAccreditedSystem(SdsBaseModel):
     nhs_product_name: Optional[str] = Field(alias="nhsproductname")
     nhs_product_version: Optional[str] = Field(alias="nhsproductversion")
     nhs_as_acf: Optional[set[str]] = Field(alias="nhsasacf")
-    nhs_as_client: Optional[set[str]] = Field(alias="nhsasclient")
+    nhs_as_client: Optional[set[str]] = Field(alias="nhsasclient", default_factory=set)
```

Comment:
Does this still make it optional? Or will it always exist as at least an empty set now?